### PR TITLE
Umbrella crate: Add polkadot-sdk-frame/?runtime

### PR DESCRIPTION
**Pull request by pgherveou**
This should make it possible to use the umbrella crate alone for templates/*/runtime crate of the repo

### CODE REVIEW

File: umbrella/Cargo.toml [hunk 1]
```diff
@@ -540,7 +540,7 @@ with-tracing = [
...
..."snowbridge-router-primitives", "snowbridge-runtime-common", "snowbridge-system-runtime-api", "sp-api", "sp-api-proc-macro", "sp-application-crypto", "sp-arithmetic", "sp-authority-discovery", "sp-block-builder", "sp-consensus-aura", "sp-consensus-babe", "sp-consensus-beefy", "sp-consensus-grandpa", "sp-consensus-pow", "sp-consensus-slots", "sp-core", "sp-crypto-ec-utils", "sp-crypto-hashing", "sp-crypto-hashing-proc-macro", "sp-debug-derive", "sp-externalities", "sp-genesis-builder", "sp-inherents", "sp-io", "sp-keyring", "sp-keystore", "sp-metadata-ir", "sp-mixnet", "sp-mmr-primitives", "sp-npos-elections", "sp-offchain", "sp-runtime", "sp-runtime-interface", "sp-runtime-interface-proc-macro", "sp-session", "sp-staking", "sp-state-machine", "sp-statement-store", "sp-std", "sp-storage", "sp-timestamp", "sp-tracing", "sp-transaction-pool", "sp-transaction-storage-proof", "sp-trie", "sp-version", "sp-version-proc-macro", "sp-wasm-interface", "sp-weights", "staging-parachain-info", "staging-xcm", "staging-xcm-builder", "staging-xcm-executor", "substrate-bip39", "testnet-parachains-constants", "tracing-gum-proc-macro", "westend-runtime-constants", "xcm-procedural", "xcm-runtime-apis"]
```
Comment (pgherveou) [inline umbrella/Cargo.toml:543]: this just adds polkadot-sdk-frame/?runtime not sure how we want to format this and whether or not this is generated by some kind of script

**Review by kianenigma — APPROVED** [review]

**Comment by kianenigma** [comment]
Thank you! cc @gupnik

File: umbrella/Cargo.toml (hunk 1 above)
Comment (bkchr) [reply umbrella/Cargo.toml:543]: This is generated by: https://github.com/paritytech/polkadot-sdk/blob/master/scripts/generate-umbrella.py

Comment (ggwpez) [reply umbrella/Cargo.toml:543]: I think you can add it here. https://github.com/paritytech/polkadot-sdk/blob/6efa1ee85ad1b5d83ec69843fce85e8cd9a4d2f5/scripts/generate-umbrella.py#L107

Comment (pgherveou) [reply umbrella/Cargo.toml:543]: Yep added it to zepter, re-ran the umbrella script it does add the dependency correctly

**Review by ggwpez — APPROVED** [review]

**Pull request merged**